### ISSUE TITLE
Bread crumb arrows showing twice

### ISSUE THREAD

**Issue by MichaelSolati** [GoogleChrome/web.dev#3676]
On a blog post a `>` appears twice, where it should only appear once:

<img width="276" alt="Screen Shot 2020-08-10 at 7 29 26 PM" src="https://user-images.githubusercontent.com/11811422/89850857-fd75cd80-db3f-11ea-9e86-7d6c955ce258.png">

**Comment by robdodson** [comment]
I _think_ this is because our service worker is causing the html and css to be out of sync.

@samthor should we land your versioned partials thing?

**Comment by robdodson** [comment]
forgot to mention—i recently updated the breadcrumb to use an inline svg in the html instead of a CSS pseudo element.
https://github.com/GoogleChrome/web.dev/pull/3653

So I think that's why you're seeing this now. If our html and css were in sync this wouldn't happen. What I've noticed is things stay out of sync sometimes. For the BLM banner I had to inline the CSS because I couldn't ever get the service worker to grab the latest styles.
